.travis.yml

```diff
@@ -131,6 +151,18 @@ install:
     git describe
     # Upgrade pip and setuptools and wheel to get as clean an install as possible.
     python -mpip install --upgrade pip setuptools wheel
+  - |
+    # Using conda to install pandas, pyzmq & ipython
```

Comment:
can we skip the tests the rely on pandas and jupyter instead?  I would rather not bring conda / conda-forge into the mix on travis if possible...

Reply:
Sure, we can skip those tests for Arm64 platform. Updated the same.